functions/src/services/GoogleSheets.ts

```diff
@@ -65,6 +66,65 @@ export default class GoogleSheets {
     return this._convertRows(values);
   }
 
+  /**
+   * Get a number of rows (start - limit) from a specific google sheet within a spreadsheet
+   * with a maximum number of rows (limit) of 1000
+   * 
+   * @param start The row to start reading from
+   * @param limit How many rows you want
+   */
+  public async getRowsByStart(start: number, limit?: number): Promise<any> {
```

Comment:
I don't expect that this function should be public.

Reply:
Ok, I replaced now the previous `getRows` with the body of the `__getRows` in this commit.
Also, made `getRowsByStart` **protected** and changed its name to  `_getRowsByStart`

Reply:
@anantakrishna